package.json

```diff
@@ -5,9 +5,10 @@
   "scripts": {
     "build": "tsc",
     "build-watch": "tsc -w",
+    "format": "prettier --write 'lib/*.{js,ts}' 'tests/*.{js,ts}'",
     "tap": "tap test/*.test.* -Rspec --timeout=180 --node-path ts-node --test-file-pattern '/\\.[tj]s$/'",
     "test": "npm run lint && npm run tap",
```

Comment:
We are linting in travis on each step then running npm test and linting again. Sounds like we are over-linting our code 😄 
https://github.com/snyk/snyk-to-html/blob/master/.travis.yml#L15

WDYT about having a separate step before test that will lint code on node version 10 and then start testing? This will speed up build time also

Reply:
Do we have an example of this somewhere? Lint should run first